src/marqo/core/models/marqo_index.py

```diff
@@ -397,6 +397,13 @@ def field_map_by_type(self) -> Dict[FieldType, List[Field]]:
                                            for field_type in FieldType}
                                   )
 
+    @property
+    def dependent_fields_names(self) -> Set[str]:
+        return self._cache_or_get('dependent_fields_names',
```

Comment:
To be clear, this is all dependent fields from all multimodal fields in one set. Is this what you want?

Reply:
Also a docstring would help